app/views/content/life-as-a-teacher/teaching-as-a-career/_what-teachers-have-to-say.html.erb

```diff
@@ -36,6 +36,12 @@
                   description: "In Ben’s second year of teaching, he took on the role of head of physics. He then moved departments and became assistant head of maths.",
                   path: "/life-as-a-teacher/teaching-as-a-career/from-teaching-internship-to-leadership-roles"
                 )) %>
+              <%= render Content::InspirationCardComponent.new(heading_tag: "h3", card:
+                OpenStruct.new(
+                  title: "Turning a tough lesson into a success",
+                  description: "Danny shares how he tackles challenging behaviour in PE lessons.",
```

Comment:
Should we put `physical education (PE)` rather than just `PE` - I'm thinking particularly of non-UK candidates who might be less familiar with this term? Plus Find uses `physical education`

But I know the GDS style guide says use either so happy for you to disagree!